modules/rest-api/pages/rest-xdcr-identify-target-cluster.adoc

```diff
@@ -0,0 +1,180 @@
+= Identifying the Target in Replication
```

Comment:
"Identifying the Target Cluster" should be changed to "Listing Incoming Replications" in the XDCR API section.